exporter/splunkhecexporter/config.go

```diff
@@ -46,6 +47,20 @@ type OtelToHecFields struct {
 	SeverityNumber string `mapstructure:"severity_number"`
 }
 
+// HecHeartbeat defines the heartbeat information for the exporter
+type HecHeartbeat struct {
+	// Interval represents the time interval for the heartbeat interval. If nothing or 0 is set,
+	// heartbeat is not enabled.
+	// A heartbeat is an event sent to _internal index with metadata for the current collector/host.
+	Interval time.Duration `mapstructure:"hec_heartbeat_interval"`
```

Comment:
Do we need to duplicate `hec_heartbeat_` if the struct is already called exposed as `hec_heartbeat`? Maybe just `interval`? Usually, we call the filed names the same as the user interface.

```suggestion
	Interval time.Duration `mapstructure:"interval"`
```

Reply:
makes sense. will change